app/components/header/header.component.html

```diff
@@ -100,10 +100,15 @@
         <li class="toolbar-Icons" onclick="revertCommit()">
             <a href="#"><img src="./assets/Revert.svg" height="40" width="48" class="add-repository-button" title="Revert"></a>
             <span class="toolbar-Text">Revert</span>
-          </li>
+        </li>
+
+        <!--Stash-->
+        <li class="toolbar-Icons" onclick="showStashModal()">
+            <a href="#"><img src="./assets/stash.png" height="40" width="48" class="add-repository-button" title="Stash"></a>
```

Comment:
Do we have this image in SVG format? I don't see it in font awesome so it would have to be custom. SVG matches the other buttons and keeps it's clarity across resolutions

Reply:
No, this was a home made image (originally a JPG) that I converted to PNG with a transparent background and inverted the colours to be consistent with others.

Should we create another issue to make this SVG?

Reply:
If we don't have one I think it's fine